### ISSUE TITLE
Is there a way to run `llvm-cov` with `nextest`?

### ISSUE THREAD

**Issue by vleni** [ipetkov/crane#559]
It is possible to call llvm-cov from `cargo-nextest`: https://nexte.st/book/test-coverage.html

Is there a way to do it with crane? I have some tests that must be run in their own processes.

**Comment by dpc** [comment]
That's probably a good question to ask on cargo-nextest github. If it's possible to do, crane will work with it 99.9%.

**Comment by vleni** [comment]
> That's probably a good question to ask on cargo-nextest github. If it's possible to do, crane will work with it 99.9%.

nextest supports it, but if I write a target like this:
```nix
        llvmCoverage = craneLibDev.cargoNextest (common // {
          inherit cargoArtifacts;
          checkPhaseCargoCommand = "cargo nextest llvm-cov --release";
        });
```
Then I get two zst files in the `result/` folder:
```
$ ls result/
target.tar.zst  target.tar.zst.prev@
```

**Comment by dpc** [comment]
> Then I get two zst files in the `result/` folder:

That looks kind of normal. crane does a incremental/layered `target` compression, so every target contains only new files.

If you're running a lcov, you probalby don't want to install the target and just copy lcov file as the result. 

Here is an example how we run in our project:
https://github.com/fedimint/fedimint/blob/df66dfae3e24465c2c0a160119bd64f91498a09d/nix/flakebox.nix#L352

Notice the `installPhaseCommand = "true";` which IIRC was a hacky way to avoid installing stuff into target and result only in `--output-path $out/lcov.info`.

I wasn't aware nextest is supported, I need to switch us to use it here as well. Thanks for letting me know!

Possilby `craneLib.cargoNextest` should default to not installing target. @ipetkov . Otherwise I'm not sure if it should support nextest via different flag or something. Last time I tried I just went with customizing `craneLib.buildPackage` manually, for better control.
